NOTICE

```diff
@@ -1,17 +1,24 @@
-CRATE PDO Adapter
-Copyright 2014-2015 CRATE Technology GmbH ("Crate")
+CrateDB PDO Adapter
+Copyright 2013-2018 Crate.IO GmbH ("Crate")
 
-Third party dependencies:
 
-=========================================================================
+Licensed to Crate.IO GmbH (referred to in this notice as "Crate") under one or
+more contributor license agreements.
 
-Copyright (2014) by Roave, LLC (http://www.roave.com)
-Special thanks go to Roave for helping to write the main parts of this project.
```

Comment:
@jodok please specifically check this removal

Reply:
authorship can be seen on the github commits. i think this credit is ok. i can send a short message to roave that we homogenized the licenses and notice files.